modules/nf-core/elprep/filter/main.nf

```diff
@@ -65,6 +65,8 @@ process ELPREP_FILTER {
     def assembly_regions_cmd = get_assembly_regions ? " --assembly-regions ${prefix}.assembly_regions.igv": ""
 
     """
+    mkdir logs
```

Comment:
Do the logs get emitted?

Reply:
Thank you!

No the logs are not emitted currently. Due to the permissions issue, the docker self hosted was failing. So I added a command to create a logs folder in the current directory. Fortunately, the tool has a `--log-path` parameter.

Reply:
Hi @SPPearce 

Should we emit the logs? Or is this good to go?

Reply:
If they are informative at all, should probably emit them

Reply:
Good idea. That's done now.